README.md

```diff
@@ -106,6 +106,34 @@ Be sure there are required dependencies installed on your computer:
 1. Open http://localhost:3000 in your browser
 1. Use `user@example.com` with the `password` to sign in.
 
+#### How to deploy on Heroku
```

Comment:
```suggestion
#### How to deploy on Heroku

ℹ️ _That is an experimental approach, at the moment it requires some tweaks in the codebase to make it work. But we're working on improving that process._
```

Let's add an explanation, because I have to make some changes in the codebase to make it work with Postgresql